src/blocks/blocks/countdown/deprecated.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { useBlockProps } from '@wordpress/block-editor';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import {
+	getIntervalFromUnix,
+	insertBetweenItems
+} from '../../helpers/helper-functions.js';
+import { omit } from 'lodash';
+
+const { attributes } = metadata;
+
+const deprecated = [ {
+	attributes: {
+		...attributes,
+		'borderRadiusTopRight': {
+			'type': 'number'
+		},
+		'borderRadiusTopLeft': {
+			'type': 'number'
+		},
+		'borderRadiusBottomRight': {
+			'type': 'number'
+		},
+		'borderRadiusBottomLeft': {
+			'type': 'number'
+		},
+		'borderRadiusType': {
+			'type': 'string',
+			'default': 'linked'
+		},
+		'borderRadius': {
+			'type': 'number'
+		}
+	},
+
+	supports: {
+		align: [ 'wide', 'full' ],
+		html: false
+	},
+
+	migrate: oldAttributes => {
+
+		const optionalUnit = x => isNumber( x ) ? x + '%' : x;
+
+		const borderRadiusBox = 'linked' === attributes.borderRadiusType ?
+			{ left: optionalUnit( attributes.borderRadius ), right: optionalUnit( attributes.borderRadius ), bottom: optionalUnit( attributes.borderRadius ), top: optionalUnit( attributes.borderRadius ) } :
+			{ left: optionalUnit( attributes.borderRadiusBottomLeft ), right: optionalUnit( attributes.borderRadiusTopRight ), bottom: optionalUnit( attributes.borderRadiusBottomRight ), top: optionalUnit( attributes.borderRadiusTopLeft ) };
+		oldAttributes.borderRadiusBox = borderRadiusBox;
+
+		return omit( oldAttributes, [ 'borderRadiusBottomLeft', 'borderRadiusTopRight', 'borderRadiusBottomRight', 'borderRadiusTopLeft', 'borderRadiusType' ]);
+	},
+
+	isEligible: ({ borderRadius }) => borderRadius && 'number' === typeof borderRadius,
+
```

Comment:
@HardeepAsrani can you tell if this part is ok? It seams that `borderRadiusBox` is not computed

Reply:
The only way to find out is to test really. You can just copy/paste old blocks from another editor and will find, really can't say much from code.